sdk/storage/README.md

```diff
@@ -139,139 +157,181 @@ For example, you can create following CORS settings for debugging. But please cu
 - Exposed headers: \*
 - Maximum age (seconds): 86400
 
-## SDK Architecture
+## Architecture
 
-The Azure Storage SDK for JavaScript provides low-level and high-level APIs. Take Blob SDK as example:
+The Azure Storage client libraries for JavaScript provides low-level and high-level APIs. Take Blob client library as example:
 
-- ServiceURL, ContainerURL and BlobURL objects provide the low-level API functionality and map one-to-one to the [Azure Storage Blob REST APIs](https://docs.microsoft.com/en-us/rest/api/storageservices/blob-service-rest-api).
+- ServiceClient, ContainerClient and BlobClient objects provide the low-level API functionality and map one-to-one to the [Azure Storage Blob REST APIs](https://docs.microsoft.com/en-us/rest/api/storageservices/blob-service-rest-api).
 
 - The high-level APIs provide convenience abstractions such as uploading a large stream to a block blob (using multiple PutBlock requests).
 
+## Examples
+
 ## Code Samples
 
 ```javascript
 const {
   BlobServiceClient,
   newPipeline,
-  SharedKeyCredential,
-  AnonymousCredential,
-  TokenCredential
+  SharedKeyCredential
 } = require("@azure/storage-blob");
 
 async function main() {
   // Enter your storage account name and shared key
-  const account = "account";
-  const accountKey = "accountkey";
+  const account = "";
+  const accountKey = "";
 
   // Use SharedKeyCredential with storage account and account key
   const sharedKeyCredential = new SharedKeyCredential(account, accountKey);
 
```

Comment:
Add comments about SharedKeyCredential is only avaiable in Node.js. 

Same with readme in blob/queue/file folder

Reply:
Good catch. Other places have been updated as well.

---
In reply to: [297670785](https://github.com/Azure/azure-sdk-for-js/pull/3744#discussion_r297670785) [](ancestors = 297670785)